src/modules/Core/default-variables.scss

```
@@ -163,7 +163,7 @@ body {
   --anp-rainbow-folder-collapse-border-custom: #00000044;
   --anp-preview-width-pct: 95%;
   --anp-preview-width-max: 800px;
-  --anp-header-margin-value: 10px;
+  --anp-header-margin-value: 4px;
```

I'd suggest using 15px since that's the default obsidian margin value if we're setting defaults.
```suggestion
  --anp-header-margin-value: 15px;
```